pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "hf_hydrodata"
-version = "1.2.7"
+version = "1.2.8"
```

Comment:
Yes. I originally used 1.3.1 as the version since this a huge change. But then I thought there is no external change at all so that might concern users since we just sent out that announcement for the new version of CW3E change. So I changed it back to just 1.2.8. I can see arguments for either way. We should all talk about this.

Reply:
I'm going to bump it to 1.3.1.